src/darwin/Framework/CHIP/MTRBaseDevice.h

```diff
@@ -196,7 +222,19 @@ extern NSString * const MTRArrayValueType;
                             attributeID:(NSNumber * _Nullable)attributeID
                                  params:(MTRReadParams * _Nullable)params
                                   queue:(dispatch_queue_t)queue
-                             completion:(MTRDeviceResponseHandler)completion;
+                             completion:(MTRDeviceResponseHandler)completion API_AVAILABLE(TODOVERSION);
+
+/**
+ * Deprecated version of readAttributePathWithEndpointID.
+ */
+- (void)readAttributeWithEndpointId:(NSNumber * _Nullable)endpointId
```

Comment:
Given we have significant amount of header documentation, I think it would be best to move the deprecated methods to the bottom of the file for clarity. I've also seen some API headers that put deprecated methods into a deprecated category.

Reply:
Yep, good idea.  I will do the moving to the bottom, and will look into the deprecated category bit.